web/src/components/Modal.tsx

```diff
@@ -21,9 +23,10 @@ const Overlay = styled.div`
   background-color: ${props => props.theme.colors.textSecondaryColor};
   opacity: 0.9;
 `
-const ModalContainer = styled.div<{ topPosition: number }>`
+
+const ModalContainer = styled.div`
   position: fixed;
-  top: ${props => props.topPosition}px;
+  top: 0;
```

Comment:
I made the modal fullscreen now to avoid flickering if opening feedback after scrolling down (look at the header): 

https://github.com/digitalfabrik/integreat-app/assets/29095487/aeed619c-c20d-4405-b0a7-2671efbd7c84